src/components/shared/bylineCard.tsx

```diff
@@ -0,0 +1,200 @@
+import React, { ReactElement, FC } from 'react';
+import { RelatedItem } from '@guardian/apps-rendering-api-models/relatedItem';
+import { Option } from '@guardian/types/option';
+import { css, SerializedStyles } from '@emotion/core';
+import { headline, textSans } from '@guardian/src-foundations/typography';
+import { remSpace } from '@guardian/src-foundations';
+import { withDefault, map, fromNullable } from '@guardian/types/option';
+import { pipe2 } from 'lib';
+import { neutral, opinion, text } from '@guardian/src-foundations/palette';
+import { Design, Display, Format } from '@guardian/types/Format';
+import { darkModeCss } from 'styles';
+import { getPillarStyles, pillarFromString } from 'pillarStyles';
+import { SvgQuote } from '@guardian/src-icons';
+import { makeRelativeDate } from 'date';
+
+interface Props {
+    relatedItem: RelatedItem;
+}
+
+const borderColor = (format: Format): SerializedStyles => {
+    return css`1px solid ${getPillarStyles(format.pillar).kicker}`
+}
+
+const listStyles = (format: Format): SerializedStyles => {
+    return css`
+        background: white;
+        margin-right: ${remSpace[3]};
+        flex: 0 0 15rem;
+        height: 100%;
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+        border-top : ${borderColor(format)};
+
+        &.fade {
+            opacity: .7;
+        }
+
+        ${darkModeCss`
+            background: ${neutral[7]};
+        `}
+    `;
+}
+
+const bylineImage = css`
+    border-radius: 50%;
+    right: 0.625rem;
+    top: 0.375rem;
+    overflow: hidden;
+    height: 8.25rem;
+    width: 8.25rem;
+    contain: paint;
```

Comment:
Fancy! What does this do?

Reply:
Had to check this myself, might have used it from dotcom:
https://developers.google.com/web/updates/2016/06/css-containment#paint_contain_paint